lib/ui.js

```diff
@@ -46,11 +49,25 @@ const printCommitLog = async repoUrl => {
 		})
 		.join('\n');
 
+	const changeLog = nextTag => 'Changes: \n' +
+		log.split('\n')
+			.map(commit => {
+				const splitIndex = commit.lastIndexOf(' ');
+				const commitMessage = util.linkifyIssues(repoUrl, commit.slice(0, splitIndex));
```

Comment:
There's a lot of duplication here from the `history` parsing (line 43). Would be good to have a generic `parseGitLog()` function that parses the Git log output into an array of objects with `.message` and `.id` properties, that is then used both here and for `history`.